core/utils/poly-look/src/react/chips/chips.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import "./chips.css";
+
+const Chips = ({ chipsContent, activeChip, onChipClick, theme = "light" }) => {
+  return (
+    <div className={`poly-theme-${theme} chips-container`}>
```

Comment:
Naming nit: Why `chips-container` and not `chips`, i.e. the exact name as the component? Also, wouldn't we want to prefix the name with `poly-`? (However, if we don't do that yet, this PR is not the right time to start I'd argue.)

Reply:
Should start good habits when importing into poly-look. Will rename to poly-chips